plugins/pom.xml

```diff
@@ -84,6 +84,7 @@
         <module>salesforce</module>
         <module>pur</module>
         <module>ms-access</module>
+        <module>rss</module>
```

Comment:
If this plugin is getting moved from kettle-engine to a plugin, this should go in the lowdeps section above, not in highdeps.